src/components/layout/nav/ToolbarMenu.tsx

```diff
@@ -9,9 +9,12 @@ import { RootPermissionsFilter } from '@/modules/permissions/PermissionsFilters'
 import { Routes } from '@/routes/routes';
 import { RootPermissionsFragment } from '@/types/gqlTypes';
 
-export const TOOLBAR_MENU_ITEMS: (NavItem<RootPermissionsFragment> & {
-  activeItemPathIncludes: string;
-})[] = [
+export const TOOLBAR_MENU_ITEMS: (Required<
```

Comment:
Let me know if you think this is too fiddly. `TOOLBAR_MENU_ITEMS` always have `path`, though it's not required on `NavItem`, so I've made it required here. But we could also just pass `|| ''` in on line 81 and 87 of this file